.github/workflows/add-item-to-project.yml

```diff
@@ -0,0 +1,55 @@
+name: 'Add Issue/PR to Project by team'
+
+on:
+  workflow_call:
+    inputs:
+      project-url:
+        description: 'URL of the GitHub Project where items should be added'
+        required: true
+        type: string
+      team-name:
+        description: 'Team name to match for PR review_requested or requested_team'
+        required: true
+        type: string
+      team-label:
+        description: 'Label that indicates the Issue/PR belongs to the team'
+        required: true
+        type: string
+    secrets:
+      GITHUB_TOKEN:
+        required: true
+
+jobs:
+  add_to_project:
+    name: 'Add to Project Board'
+    runs-on: ubuntu-latest
+
+    # Ensure we have permissions to read issues/PRs and create project issues
+    # (some repos may need 'contents: read', 'issues: write', 'pull-requests: write', etc.)
+    permissions:
+      contents: read
+      issues: write
+      pull-requests: write
+      # If you are using "classic" Projects, also add: projects: write
+
+    steps:
+      - name: Add item to project board
+        uses: actions/add-to-project@244f685bbc3b7adfa8466e08b698b5577571133e
+        if: |
+          (github.event_name == 'pull_request' &&
+            (
+              github.event.requested_team.name == inputs.team-name ||
```

Comment:
I know this is in the `core` workflow, but what is this line intended to do? I couldn't find a `requested_team` event in the docs.

Reply:
It will add the PR to our board if the team is requested to review the PR. The docs can be found here: https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request

Reply:
Ah, thanks, I missed the bit about `github.event.requested_team` in the docs. Thanks.